### PR TITLE
Create dlvr.it.xml

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/dlvr.it.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		<test url="http://static.dlvr.it/font-awesome/css/font-awesome.min.css" />
+	<target host="stories.dlvr.it" />
+	<target host="support.dlvr.it" />
+
+	<target host="dlvrit.com" />
+	<target host="www.dlvrit.com" />
+	<target host="api.dlvrit.com" />
+		<test url="http://api.dlvrit.com/1/help" />
+	<target host="app.dlvrit.com" />
+	<target host="blog.dlvrit.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/dlvr.it.xml]: dev.dlvrit.com

File: src/chrome/content/rules/dlvr.it.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<!--
+	Invalid certificate:
+		blog.dlvr.it
+		support.dlvr.it
+		feeds.promotedstories.com
```
Comment (J0WI) [inline src/chrome/content/rules/dlvr.it.xml]: mailtrack.dlvr.it (m)

Comment (Bisaloo) [inline src/chrome/content/rules/dlvr.it.xml]: It doesn't look like a functional domain, even over plain `http`.

Comment (J0WI) [inline src/chrome/content/rules/dlvr.it.xml]: This host tracks clicks to (external) links. On $ it has no content.

**Pull request merged**